src/Extension/Resources/config/task/RunInstallTask.yaml

```
@@ -1,8 +1,8 @@
 ---
-id: "project:install:run"
+id: "install:project:run"
```

If it decided to change a command name, let's name it properly, keeping install and run in the same line just doesn't male sense.  
```suggestion
id: "spryker:installer:run"
or
id: "spryker:project-installer:run"
```